platform_design/lib/main.dart

```diff
@@ -20,6 +20,9 @@ class MyAdaptingApp extends StatelessWidget {
         // Use the green theme for Material widgets.
         primarySwatch: Colors.green,
       ),
+      darkTheme: ThemeData.dark().copyWith(
+        scaffoldBackgroundColor: Colors.black,
```

Comment:
I'm not sure that changing the default scaffold background to black is a good idea.

* It can cause the background of the scaffold to merge with the black background of the back/home buttons at the bottom of the screen.
* One of the general rules of thumb for design is to avoid using pure black.
  - See https://ianstormtaylor.com/design-tip-never-use-black/
* In general, we try to have practice good code size economy in every sample, since the longer a sample is, the more effort a reader has to spend learning from it. For that reason, if an addition of new code isn't absolutely necessary, it's generally best to take it out.